src/yb/tools/ts-cli.cc

```diff
@@ -194,15 +208,22 @@ TsAdminClient::~TsAdminClient() {
 Status TsAdminClient::Init() {
   CHECK(!initted_);
 
-  HostPort host_port;
-  RETURN_NOT_OK(host_port.ParseString(addr_, tserver::TabletServer::kDefaultPort));
-  messenger_ = VERIFY_RESULT(MessengerBuilder("ts-cli").Build());
-
-  rpc::ProxyCache proxy_cache(messenger_.get());
-
-  generic_proxy_.reset(new server::GenericServiceProxy(&proxy_cache, host_port));
-  ts_proxy_.reset(new TabletServerServiceProxy(&proxy_cache, host_port));
-  ts_admin_proxy_.reset(new TabletServerAdminServiceProxy(&proxy_cache, host_port));
+  if(!certs_dir_.empty()) {
```

Comment:
In other place of code enabling encryption is done base on `use_node_to_node_encryption` or/and `use_client_to_server_encryption`. It these flags are set but `certs_dir` is not default value for `certs_dir` is used.